packages/documentation/plugins/mermaid-svg-remover/gatsby-node.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs');
+const path = require('path');
+
```

Comment:
This will remove all SVG files generated by the mermaid.cli. We have no way to track if a mermaid graph is removed from a document. This will reset it so it will only have graphs that are on current pages.